src/main/java/com/libertymutualgroup/herman/aws/ecs/broker/s3/S3BucketEncryption.java

```diff
@@ -1,71 +0,0 @@
-/*
```

Comment:
Any reason this file was deleted or was this another merge issue?

Reply:
This was not being used anywhere, so removed it. By the name I thought it was a model class at first tbh, I don't have any problems adding this back in but should be something like S3BucketEncryptionHandler. + @jelazos7

Reply:
Ah yeah, looks like this functionality got rolled up into the S3 broker itself, so good call here